packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts

```diff
@@ -47,7 +47,7 @@ export default ({
     case manualEntryTypeText:
       if (!manualUrl) {
         warn(
-          `Bad content for ${contentTypeId}: DestinationType is ${manualEntryTypeText}, but no URL has been entered`
+          `Bad content for ${contentTypeId} with Content ID ${contentReference?.sys.id}: DestinationType is ${manualEntryTypeText}, but no URL has been entered`
```

Comment:
contentReference is wrong here, what we want is the ID of the ElementLink item, which is the one that has the missing fields. contentReference is a field within this item that may or may not have an item in it.  If the ID of the ElementLink item is not available in LinkParser, we may have to add another argument and pass it in from the caller.

Reply:
@prceasar Gotcha, thx! Just curious then, is that ID the same `id` listed here?
https://github.com/last-rev-llc/lastrev-toolkit/blob/ef881b72502bb75a9869aba3ee64ae01dedd8925/packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts#L31
..which is passed into the caller here:
https://github.com/last-rev-llc/lastrev-toolkit/blob/98a1c9f3c44cf4ace22e969a60872fb9db315a8a/packages/lastrev-adapter-contentful/src/contentfulAdapter/contentfulAdapter.ts#L41

Reply:
@altruity Yes, I believe that's it.